code/server.py

```diff
@@ -0,0 +1,53 @@
+from os import path
+
+from flask import Flask, g, session, redirect, request, url_for, jsonify, render_template, send_from_directory, escape
```

Comment:
Why not `import flask`, and use `flask.escape` in the code? What should I think, when I see a legendary `g` variable somewhere?